### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by nicosampler**

### CODE REVIEW

File: pages_/my-shop.tsx [hunk 1]
```diff
@@ -149,10 +116,11 @@ const MyShop = ({ isLoggedIn, shopId, encodedShopId }: Props) => {
   }, [isLoggedIn, shopId]);
 
   if (error) {
+    Sentry.captureException(error);
```
Comment (ramirotw) [inline pages_/my-shop.tsx:119]: we are catching global errors in the ErrorBoundary component and there we are using the captureException call

File: graphql/issuedNumber/resolver.ts [hunk 2]
```diff
@@ -115,15 +115,15 @@ const IssuedNumberResolver = {
...
-      }
+      // const todayAppointments = turns.filter(
+      //   (a) => compareAsc(a.createdAt, startOfDay(new Date())) >= 0
+      // );
+      // if (todayAppointments.length >= 5) {
+      //   return new ApolloError(
+      //     'Today turns quota exceeded',
+      //     'TODAY_TURNS_QUOTA_EXCEEDED'
+      //   );
+      // }
```
Comment (ramirotw) [inline graphql/issuedNumber/resolver.ts:126]: are we going to leave this disabled?

File: pages_/my-shop.tsx (hunk 1 above)
Comment (nicosampler) [reply pages_/my-shop.tsx:119]: gotcha, shall we remove all the occurrences of this?

File: graphql/issuedNumber/resolver.ts (hunk 2 above)
Comment (nicosampler) [reply graphql/issuedNumber/resolver.ts]: done

File: pages_/my-shop.tsx (hunk 1 above)
Comment (nicosampler) [reply pages_/my-shop.tsx]: done

**Pull request merged**